OrangeHRMTests/PageObjects/LoginPage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.Data;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class LoginPage : BasePage
+    {
+        private MyWebElement UserNameField = new MyWebElement(By.XPath("//*[contains(@class, 'form-row')]//*[contains(@name, 'username')]"));
+        private MyWebElement PassWordField = new MyWebElement(By.XPath("//*[contains(@class, 'form-row')]//*[contains(@name, 'password')]"));
+        private MyWebElement InvalidMessageTextElement = new MyWebElement(By.XPath("//*[contains(@class, 'login-error')]//*[contains(@class, 'alert-content-text')]"));
+        private MyWebElement ForgotPasswordButton = new MyWebElement(By.XPath("//*[contains(@class, 'login-forgot')]/p"));
+        private MyWebElement ResetPasswordMessageTextElement = new MyWebElement(By.XPath("//*[contains(@class, 'forgot-password-title')]"));
+
+        public string GetCurrentPageUrl() => WebDriverFactory.Driver.Url;
+
+        public void EnterValidUsername() => UserNameField.SendKeys(TestSettings.Username);
```

Comment:
What will you do if you need to create a test where you have to input invalid creds? Create a new method like input EnterInvalidUserName?
I would better make method: InputUserName(string userName) and put TestSettings.UserName as parameter in test. The same for password

Reply:
Fixed